lib/src/sync/target_locator.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'alert.dart';
+import 'common.dart';
+import 'web_driver.dart';
+import 'window.dart';
+
+class TargetLocator {
+  final WebDriver _driver;
+  final Resolver _resolver;
+
+  TargetLocator(this._driver) : _resolver = new Resolver(_driver, '');
+
+  /// Change focus to another frame on the page.
+  /// If [frame] is a:
+  ///   [int]: select by its zero-based index
```

Comment:
Not 100% certain, but I believe that W3C removed support for int and string here.